lib/validator/sfValidatorFile.class.php

```diff
@@ -263,7 +263,9 @@ protected function guessFromFileBinary($file)
   {
     ob_start();
     //need to use --mime instead of -i. see #6641
-    passthru(sprintf('file -b --mime %s 2>/dev/null', escapeshellarg($file)), $return);
+    $cmd = 'file -b --mime %s -- 2>/dev/null';
```

Comment:
The path need to be placed after the double dash.

Reply:
Well spotted thanks 👍